monailabel/transform/post.py

```diff
@@ -124,6 +134,16 @@ def __call__(self, data):
                 resizer = Resize(spatial_size=spatial_size, mode=self.mode[idx])
                 result = resizer(result, mode=self.mode[idx], align_corners=self.align_corners[idx])
 
+            if self.invert_orient:
+                # Undo Orientation
+                orig_affine = meta_dict["original_affine"]
+                orig_axcodes = nib.orientations.aff2axcodes(orig_affine)
+                inverse_transform = Orientation(axcodes=orig_axcodes)
+                # Apply inverse
+                logging.info(f"THIS IS THE SHAPEEE: {len(result.shape)}")
```

Comment:
Is the "SHAPEEE" preferred here? Rocky logs :)

Reply:
hahahaha great catch! Just updated it